package.json

```diff
@@ -21,13 +21,29 @@
   },
   "license": "MIT",
   "dependencies": {
-    "bluebird": "^3.5.5",
+    "@aws-sdk/client-cloudformation": "^3.490.0",
+    "@aws-sdk/client-cloudtrail": "^3.490.0",
+    "@aws-sdk/client-config-service": "^3.490.0",
+    "@aws-sdk/client-ec2": "^3.496.0",
+    "@aws-sdk/client-guardduty": "^3.496.0",
+    "@aws-sdk/client-iam": "^3.490.0",
+    "@aws-sdk/client-kinesis": "^3.496.0",
+    "@aws-sdk/client-kms": "^3.496.0",
+    "@aws-sdk/client-lambda": "^3.496.0",
+    "@aws-sdk/client-organizations": "^3.497.0",
+    "@aws-sdk/client-rds": "^3.497.0",
+    "@aws-sdk/client-redshift": "^3.496.0",
+    "@aws-sdk/client-s3": "^3.490.0",
+    "@aws-sdk/client-sqs": "^3.496.0",
+    "@aws-sdk/client-sts": "^3.496.0",
+    "@aws-sdk/util-retry": "^3.374.0",
     "lodash.isequal": "^4.5.0"
   },
   "devDependencies": {
     "@conformity/commitlint-config-conformity": "^1.2.3",
     "aws-sdk": "^2.912.0",
```

Comment:
Hi @NicholasValiente Is the `aws-sdk` v2 required?

Reply:
no it isnt, thanks for spotting that i have removed it and pushed the change